ci-release.sh

```diff
@@ -20,10 +24,32 @@ fi
 git config user.name "CI release"
 git config user.email "ci-test@release.com"
 
+# Ensure the remote repository 'upstream' is set to the correct URL
+if git remote | grep -iq 'upstream'; then
+    current_url=$(git remote get-url upstream)
+    if [ "$current_url" != "$REPO_URL" ]; then
+        git remote set-url upstream $REPO_URL
+    fi
+else
+    git remote add upstream $REPO_URL
+fi
+
 # Ensure working on the latest main
 git fetch upstream
 git checkout upstream/main
 
 # Tagging the release
 git tag -a "v${VERSION}" -m "Release v${VERSION}"
 git push upstream "v${VERSION}"
+
+# Ensure GITHUB_TOKEN is set
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "Error: GITHUB_TOKEN is not set."
+    exit 1
+fi
+
+# Export the GITHUB_TOKEN for use by goreleaser
+export GITHUB_TOKEN
```

Comment:
should it be `export GITHUB_TOKEN=xxxx`?

Reply:
Actually we don't have to specify `GITHUB_TOKEN` and `make release` in the script since they are defined in the [CI configuration](https://github.com/xiaoyu74/release/blob/86ce2be6ae6ba40c4b4547cb8675d0d8cb513d21/ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml#L57-L61) already. 

I will update the script to remove those 2.